### ISSUE TITLE
question about the training process

### ISSUE THREAD

**Issue by sakura2233565548** [fengyang0317/unsupervised_captioning#23]
Hi Yang:
   I use the code that you release in the github, and I run the train code with the parameter: two GPUs,
batch_size 256, lr_rate 0.0005, now training 26000 batch_size, but the found_object is so low,  about 0.14. I observe your training process and your found_object is increasing so high after 14k batch_size,
so can you have some answers for this symptom? I'd appreciate it if you can help solve this problem!

**Comment by fengyang0317** [comment]
Are you using the MSCOCO dataset?
Have you tried the data files in https://drive.google.com/drive/folders/1ol8gLj6hYgluldvdj9XFKm16TCqOr7EE?

**Comment by sakura2233565548** [comment]
> Are you using the MSCOCO dataset?
> Have you tried the data files in https://drive.google.com/drive/folders/1ol8gLj6hYgluldvdj9XFKm16TCqOr7EE?

yes,I have checked the dataset ,mscoco 2014, No problem~So I am confused with this problem.

**Comment by fengyang0317** [comment]
Did you generate the training input by yourself?
You may try to use the input files from google drive.

**Comment by sakura2233565548** [comment]
> Did you generate the training input by yourself?
> You may try to use the input files from google drive.

I didn't use the input files that you release on google drive~I use the code to preprocess the image dataset by myself.
